app/models/concerns/allinson_flex/dynamic_solr_document.rb

```diff
@@ -25,7 +25,8 @@ def load_allinson_flex
               attribute(
                 prop.name,
                 # if the property is singular, make it so
-                prop.cardinality_maximum == 1 ? Hyrax::SolrDocument::Metadata::Solr::String : Hyrax::SolrDocument::Metadata::Solr::Array,
+                # WIP(alishaevn): don't base this on cardinality
+                prop.multi_value ? Hyrax::SolrDocument::Metadata::Solr::Array : Hyrax::SolrDocument::Metadata::Solr::String,
```

Comment:
I'm not actually sure what should be an array and what should be a string by default. don't think this is defined anywhere in samvera. so I made the spreadsheet below from scratch and intended to import it and basically wait until hyrax told me that it expected one thing but I gave it another.

not the most efficient. but I'm unsure how else to go about it.
[all_metadata.csv](https://github.com/samvera-labs/allinson_flex/files/9428367/all_metadata.csv)